src/main/java/net/mcreator/generator/GeneratorWrapper.java

```diff
@@ -89,6 +89,10 @@ public String getElementPlainName(String elementName) {
 				.replace(".legs", "").replace(".boots", "").replace(".bucket", "");
 	}
 
+	public String getElementClassName(String elementName) {

```

Comment:
This should be implemented in freemarker instead, as this depends on the generator template files and how this generator names the files